src/components/constants/Items.ts

```diff
@@ -408,6 +408,226 @@ export class Items {
         "Town Map": {
             "effect": "Shows your position in the Pokemon World",
             "category": "Key"
-        }
+        },
+        "HM01": {
+            "effect": "Cut",
```

Comment:
No, you need to handle them as arrays. We have to assume there will be some time in the future that weird names will happen. At the very least mods will want it.